src/debug-tools/dt.cpp

```diff
@@ -0,0 +1,31 @@
+#include <debug-tools/dt.hpp>
+
+#include "logger.hpp"
+
+using namespace dt;
+
+class debugTools::debugToolsImpl {
+public:
+  explicit debugToolsImpl()
+  : m_logger("Logger", true)
+  {};
+
+  ~debugToolsImpl() = default;
+
+  void draw() {
+    m_logger.draw();
+  }
+
+private:
+  logger m_logger;
+};
+
+debugTools::debugTools()
+: m_pimpl(new debugToolsImpl)
```

Comment:
To samo pytanie co w linii 10. Nie do końca rozumiem składnię klasa::fun():fun?()